src/data/linux-quiz.ts

```diff
@@ -821,6 +821,16 @@ const linuxQuiz = [
       "Vim stands for Vi IMproved and it is a popular open-source text editor.",
     Link: "https://www.freecodecamp.org/news/learn-vim-beginners-tutorial/",
   },
+  {
+    Question: "Which Linux Desktop Environment is a doppelganger of Windows?",
+    Answer: "KDE Plasma",
+    Distractor1: "GNOME",
+    Distractor2: "Xfce",
+    Distractor3: "Mate",
+    Explanation:
+      "KDE is an international free software community that develops free and open-source software. Its UI resembles Windows.",
```

Comment:
Let's reword this because it is to similar to this wikipedia article. In the future, please avoid direct quotes from 3rd party resources. 
https://en.wikipedia.org/wiki/KDE

```suggestion
      "KDE is a popular desktop environment which resembles the Windows user interface.",
```